### ISSUE TITLE
[Suggestion] System Preview

### ISSUE THREAD

**Issue by ZachZent** [tsunamayo/Starship-EVO#3142]
I know that star system view is planned, but in addition to that something like this would be nice so you don't have to keep entering and exiting systems just to get the basic layout. A small preview of the planets and star in the system. The amount of information is variable. It can just be grey dots so the player only knows the total number of planets or more complicated such as other structures (rings), planet "type"/color, planet name when the cursor is over it, factional claims, etc. Probably best to keep it minimal so players still have to enter to get the majority of information, but the options are there.

![](https://cdn.discordapp.com/attachments/406664114249793536/772162766072053770/unknown.png)

Also now the screen is symmetrical and when the galaxy is fully zoomed out the black boarders will be covered on both sides.

**Comment by tsunamayo** [comment]
Hum not sure about this, I think it could kill a bit the fun of exploration. On the other side I could also bind this to gameplay, with map to be found / buy, or just to simply remember from where you went. Thx

**Comment by ZachZent** [comment]
Perhaps I was a bit too ambitious when writing this. I absolutely don't want to discourage exploration and much of this info from the offset would. I was just thinking something similar to NMS where clicking on a system gives you some data in a way that would create symmetry. I was thinking a bit too much into the visuals as it would be nice to see at the very least how many planets are in the system, but this can also be displayed in text. Just a bit of convenient info for the player, especially considering looking back to a system you already visited and needing to zoom into the system just to get some basic data.

Maybe if keeping with the visuals the planets would start as grey orbs simply representing the number of planets in the system. I don't think knowing the number of planets would discourage exploration. Once you discover the planets, the grey orb will change to its texture to the planets texture and hovering over it will give you the basic data your discovered. That way when searching around on the map, it removes a little bit on inconvenience if you are looking for a planet you have visited for some resource or something.

![](https://cdn.discordapp.com/attachments/414026472047640576/772711394298036274/1200px-NMS1dot3GalaxyMap.png)

![](https://static.wikia.nocookie.net/nomanssky_gamepedia/images/1/1a/Galaxy_Map_Explanation.jpg/revision/latest?cb=20201015203207.png)

**Comment by tsunamayo** [comment]
Yes I agree I plan to show more info, as stuff are added for gameplay for sure I will add threat level, diplomacy, ressources. Cheers
